test/TestServerProjectsLowLevel/body-array/Generated/ArrayClient.cs

```diff
@@ -18,13 +18,12 @@ public partial class ArrayClient
     {
         private const string AuthorizationHeader = "Fake-Subscription-Key";
         private readonly AzureKeyCredential _keyCredential;
-
         private readonly HttpPipeline _pipeline;
         private readonly ClientDiagnostics _clientDiagnostics;
         private readonly Uri _endpoint;
 
         /// <summary> The HTTP pipeline for sending and receiving REST requests and responses. </summary>
-        public virtual HttpPipeline Pipeline { get => _pipeline; }
```

Comment:
It's a bit challenging to review PRs that include new features as well as small code-style changes, as the code-style changes touch many files, so it's hard to know which files are relevant to the feature work, and not cosmetic updates.  It would be helpful if future feature PRs could be separated out from code-style changes.